package.json

```diff
@@ -7,8 +7,10 @@
     "@testing-library/react": "^11.2.7",
```

Comment:
- [ ] It would be better if we can start the application after installing the packages
`npm i`
`npm start` returns the error in the screenshot
I am unable to approve the project because of that
![Screenshot from 2021-09-17 21-23-27](https://user-images.githubusercontent.com/46388113/133855306-6aca597d-233c-43aa-aab5-293b074987cd.png)

Reply:
Added instruction into README on the 'Getting started' section.